src/elements/Input/Input.jsx

```diff
@@ -51,6 +51,7 @@ export default {
         <input
           value={this.value}
           onInput={e => this.$emit('input', e.target.value)}
+          onBlur={e => this.$emit('blur', e)}
```

Comment:
You also need to add `blur` to events [here](https://github.com/Semantic-UI-Vue/Semantic-UI-Vue/pull/190/files#diff-491107f93fb34e2f8b674856a3224d6bR25)

Reply:
Ah whoops sorry, added now